inheritance/lib/src/main/java/inheritance/Restaurant.java

```diff
@@ -39,14 +44,15 @@ public String getPriceCategoryString() {
                 return "Free";
         }
     }
+
     @Override
     public String toString() {
         return "Restaurant{" +
                 "name='" + name + '\'' +
                 ", stars=" + stars +
                 ", priceCategory=" + priceCategory +
                 '}';
-        //return name + " - " + stars + " Stars - " + getPriceCategoryString();
-        //return name + " - " + stars + " Stars";
```

Comment:
Methods can only have one `return` statement. If you want to have the option of returning strings in multiple formats, you could either: 

- write multiple methods (something like `toStringShort`, `toStringMed`, `toStringLong`), or
- pass in a variable to your toString method to tell it which version of the string you want to return (for instance, `toString(String length)` could take in a `length` variable that you could use in a switch statement to determine which string should be returned.